### PR TITLE
SC-5734 Bug Fix for participants pop up (Long names are wrongly dispalyed)

### PR DESCRIPTION
**Pull request by abralurrahman**
# Description

## Links to Tickets or other pull requests

Base links to copy
- https://ticketsystem.schul-cloud.org/browse/SC-5734

## Screenshots of UI changes

Participants

Update:
Test pop up

 

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: static/scripts/coursesOverview.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ $(document).ready(function () {
                     if (member.displayName) {
                         courseMembers = courseMembers + '<li>' + member.displayName + '</li>';
                     } else {
-                        courseMembers = courseMembers + '<li>' + member.firstName + ' ' + member.lastName + '</li>';
+                        courseMembers = courseMembers + '<li>' + member.firstName + '<br>' + '&nbsp;&nbsp;&nbsp;&nbsp;' + member.lastName + '</li>';
```
Comment (ABialoskorski) [inline static/scripts/coursesOverview.js:23]: why are the &nbsp;&nbsp;&nbsp;&nbsp; needen, when we have br?

Comment (abralurrahman) [reply static/scripts/coursesOverview.js:23]: for the spacing. after break it starts directly from the bottom of the number. adding nbsp 4 times solved the issue.

**Review by ABialoskorski — CHANGES_REQUESTED** [review]
please use css for that

File: static/scripts/coursesOverview.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (abralurrahman) [reply static/scripts/coursesOverview.js:23]: Found another simple solution which solves most of it. may be.! ;)

**Review by DanielRieckhof — APPROVED** [review]

**Review by monaweitzenberg — APPROVED** [review]

**Review by ABialoskorski — APPROVED** [review]

**Comment by ABialoskorski** [comment]
Retry DeepCode

**Pull request merged**